integration/dataservice-hosting-tests/tests-integration/tests/src/test/java/org/wso2/ei/dataservice/integration/test/samples/JSONPayloadSampleTestCase.java

```diff
@@ -0,0 +1,154 @@
+        String postInsertPaymentPayload ="{\n" +
+                "\"employees\":{\n" +
+                    "\"employee\":[\n" +
+                        "{\n" +
+                        "\"employeeNumber\" :53,\n" +
+                        "\"lastName\": \"Sangeeth\",\n" +
+                        "\"firstName\": \"Karunaratne\",\n" +
+                        "\"email\": \"sangeeth@wso2.com\",\n" +
+                        "\"salary\": 18400.00 \n" +
+                        "},\n" +
+                        "{\n" +
+                        "\"employeeNumber\" :101,\n" +
+                        "\"lastName\": \"Smitth\",\n" +
+                        "\"firstName\": \"Will\",\n" +
+                        "\"email\": \"will@smith.com\",\n" +
+                        "\"salary\": 15500.00\n" +
+                        "}\n" +
+                      "]\n" +
+                    "}\n" +
+                "}";
+        getHttpResponse(serviceEndPoint + "employee_batch_req", "POST", postInsertPaymentPayload);
+        String response_employee1 = getHttpResponse(serviceEndPoint + "employee/53", "GET", null);
+        String response_employee2 = getHttpResponse(serviceEndPoint + "employee/101", "GET", null);
+        Assert.assertTrue(response_employee1.contains("{\"employees\":{\"employee\":[{\"lastName\":\"Sangeeth\",\"firstName\":\"Karunaratne\",\"salary\":18400.0}]}}"),
+                        "Batch request POST method failed");
+        Assert.assertTrue(response_employee2.contains("{\"employees\":{\"employee\":[{\"lastName\":\"Smitth\",\"firstName\":\"Will\",\"salary\":15500.0}]}}"),
+                        "Batch request POST method failed");
+    }
+
+    private String getHttpResponse(String endpoint, String requestMethod, String payload) {
+        StringBuilder jsonString = new StringBuilder();
+        BufferedReader br = null;
+        try {
+            String line;
+            URL url = new URL(endpoint);
+            HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+            connection.setDoOutput(true);
+            connection.setRequestProperty("charset", "UTF-8");
+            connection.setReadTimeout(10000);
+            connection.setRequestMethod(requestMethod);
+            connection.setRequestProperty("Accept", "application/json");
+            if (null != payload) {
+                connection.setRequestProperty("Content-Type", "application/json");
+                connection.setRequestProperty("Content-Length", String.valueOf(payload.length()));
+                DataOutputStream writer = new DataOutputStream(connection.getOutputStream());
+                writer.write(payload.getBytes());
+                writer.close();
+            }
+            br = new BufferedReader(new InputStreamReader(connection.getInputStream()));
+            while (null != (line = br.readLine())) {
+                jsonString.append(line);
+            }
+            connection.disconnect();
```

Comment:
Don't we have to do this in finally block?

Reply:
fixed